svf/include/SVFIR/SVFVariables.h

```diff
@@ -68,10 +69,23 @@ class SVFVar : public GenericPAGNodeTy
     SVFStmt::KindToSVFStmtMapTy InEdgeKindToSetMap;
     SVFStmt::KindToSVFStmtMapTy OutEdgeKindToSetMap;
     bool isPtr;	/// whether it is a pointer (top-level or address-taken)
+    bool ptrInUncalledFun;  ///< true if this pointer is in an uncalled function
```

Comment:
Could we use method for these two flags:

    bool isPtr;	/// whether it is a pointer (top-level or address-taken)
    bool ptrInUncalledFun;  ///< true if this pointer is in an uncalled function